crates/astria-conductor/src/sequencer/client.rs

```diff
@@ -38,7 +39,10 @@ impl SequencerGrpcClient {
         let uri: Uri = sequencer_uri
             .parse()
             .wrap_err("failed parsing provided string as Uri")?;
-        let endpoint = Endpoint::from(uri.clone());
+        let mut endpoint = Endpoint::from(uri.clone());
+        if uri.scheme() == Some(&http::uri::Scheme::HTTPS) {
```

Comment:
Is this necessary? Would setting the tls config preclude us from using http? I think we should be able to do without?

Either way, if this does blockhttp, I think I'd prefer to have ane explicit flag that deactivates tls so that one doesn't accidentally provide http instead of https and risk exposing the service to an unchecked endpoint.